common/src/main/java/com/wynntils/models/gear/GearInfoRegistry.java

```diff
@@ -379,39 +379,34 @@ private Optional<String> parseQuest(JsonObject json) {
         }
 
         private FixedStats parseFixedStats(JsonObject json) {
-            int healthBuff = JsonUtils.getNullableJsonInt(json, "health");
-            List<Pair<Skill, Integer>> skillBonuses = parseSkillBonuses(json);
+            JsonObject baseStats = JsonUtils.getNullableJsonObject(json, "base");
+            JsonObject identifications = JsonUtils.getNullableJsonObject(json, "identifications");
+
+            int healthBuff = JsonUtils.getNullableJsonInt(baseStats, "health");
+            List<Pair<Skill, Integer>> skillBonuses = parseSkillBonuses(identifications);
             String attackSpeedStr = JsonUtils.getNullableJsonString(json, "attackSpeed");
             Optional<GearAttackSpeed> attackSpeed = Optional.ofNullable(GearAttackSpeed.fromString(attackSpeedStr));
 
-            List<GearMajorId> majorIds = parseMajorIds(json);
-            List<Pair<DamageType, RangedValue>> damages = parseDamages(json);
-            List<Pair<Element, Integer>> defences = parseDefences(json);
+            Optional<GearMajorId> majorIds = parseMajorIds(json);
```

Comment:
How come majorIds is not a list anymore? Can we ever only have a single major ID? Was this always the case but the old code was overly general?

Reply:
I am not sure if the old API had provided a list, or if this was just a design decision, but the new API's data structure quite clearly suggest that an item has no intention of having multiple major ids.